Tasks/Xcode/xcode.ts

```diff
@@ -231,11 +231,17 @@ async function run() {
             xcb.pipeExecOutputToTool(xcPrettyTool);
         }
 
-        //--- Xcode Build ---
-        await xcb.exec();
+        let xcodeTaskErrors;
+        try {
+            //--- Xcode Build ---
+            await xcb.exec();
+        } catch (err) {
+            xcodeTaskErrors = err;
+            tl.debug('Xcode task failed: ' + err);
```

Comment:
Should we make this a warning instead? If `xcodebuild test` and `tp.publish` both fail, currently only the publish failures will be surfaced.